package.json

```diff
@@ -30,25 +31,20 @@
     "lib/"
   ],
   "devDependencies": {
-    "@babel/cli": "^7.8.4",
-    "@babel/core": "^7.8.4",
+    "@babel/cli": "^7.22.9",
+    "@babel/core": "^7.22.9",
     "@launchpadlab/babel-preset": "^2.1.1",
-    "@launchpadlab/eslint-config": "^2.7.0",
-    "documentation": "^12.1.4",
-    "eslint": "^6.8.0",
-    "husky": "^4.2.3",
-    "jest": "^25.1.0",
+    "@launchpadlab/eslint-config": "^3.0.1",
+    "documentation": "^14.0.0",
```

Comment:
The only change that affected this library was the change from `-` to `*` for bullet points